Xplat/src/main/java/vazkii/botania/common/block/BotaniaBlocks.java

```diff
@@ -940,6 +940,17 @@ protected Projectile getProjectile(@NotNull Level world, @NotNull Position pos,
 		behavior = DispenserBlockAccessor.getDispenserRegistry().get(Items.GLASS_BOTTLE);
 		DispenserBlock.registerBehavior(Items.GLASS_BOTTLE, new EnderAirBottlingBehavior(behavior));
 
+		behavior = new GrassSeedsBehavior();
+		DispenserBlock.registerBehavior(BotaniaItems.grassSeeds, behavior);
+		DispenserBlock.registerBehavior(BotaniaItems.podzolSeeds, behavior);
+		DispenserBlock.registerBehavior(BotaniaItems.mycelSeeds, behavior);
+		DispenserBlock.registerBehavior(BotaniaItems.drySeeds, behavior);
+		DispenserBlock.registerBehavior(BotaniaItems.goldenSeeds, behavior);
+		DispenserBlock.registerBehavior(BotaniaItems.vividSeeds, behavior);
+		DispenserBlock.registerBehavior(BotaniaItems.scorchedSeeds, behavior);
+		DispenserBlock.registerBehavior(BotaniaItems.infusedSeeds, behavior);
+		DispenserBlock.registerBehavior(BotaniaItems.mutatedSeeds, behavior);
+
```

Comment:
how about something like:

```
Item[] seeds = { BotaniaItems.grassSeeds, ...};
for (Item seed : seeds) {
  DispenserBlock.registerBehavior(item, behavior);
}
```
to avoid repeating the method call over and over